spec/lib/puppet_spec/settings.rb

```diff
@@ -11,6 +11,7 @@ module PuppetSpec::Settings
     :confdir      => { :type => :directory, :default => "test", :desc => "confdir" },
     :codedir      => { :type => :directory, :default => "test", :desc => "codedir" },
     :vardir       => { :type => :directory, :default => "test", :desc => "vardir" },
+    :publicdir    => { :type => :directory, :default => "test", :desc => "public" },
```

Comment:
`:desc => "publicdir"` seems to be the convention?

Reply:
Keyboard must've died on me there 😂